bench/workgen/workgen.h

```diff
@@ -437,34 +437,34 @@ struct Transaction {
 // To prevent silent errors, this class is set up in Python so that new
 // properties are prevented, only existing properties can be set.
 struct WorkloadOptions {
-    int max_latency;
-    bool report_enabled;
-    std::string report_file;
-    int report_interval;
-    int run_time;
-    int sample_interval_ms;
-    int sample_rate;
-    int max_idle_table_cycle;
-    std::string sample_file;
-    int warmup;
-    double oldest_timestamp_lag;
-    double stable_timestamp_lag;
-    double timestamp_advance;
-    bool max_idle_table_cycle_fatal;
-    /* Dynamic create/drop options */
+    int background_compact;
     int create_count;
     int create_interval;
+    /* Dynamic create/drop options */
```

Comment:
Previously this occurred before `create_(count|interval)`.

Reply:
fixed [Moved a comment in that was in the wrong spot and re-grouped variable…](https://github.com/wiredtiger/wiredtiger/pull/9970/commits/ed13ab8cd5f2bb557aea4ae4908f8d520daab6ba)